src/app/register/register.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RegisterComponent } from './register.component';
+
+describe('RegisterComponent', () => {
+  let component: RegisterComponent;
+  let fixture: ComponentFixture<RegisterComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ RegisterComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RegisterComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Should we require unit tests before we allow our colleagues merge/submit code? Or should unit tests be their own pull requests/branches or something?

Reply:
We definitely can. This branch does not have anything to test, although we can add it to other branches.

Reply:
My 2 cents: our app is pretty simple and in my mind if anything fails it will render the app non-functional; unit testing through use if you will.